packages/cobs/cobs.0.1.1/opam

```diff
@@ -0,0 +1,41 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Consistent Overhead Byte Stuffing (COBS) encoding and decoding"
+description:
+  "A lightweight Consistent Overhead Byte Stuffing (COBS) encoder and decoder written in pure OCaml with no dependencies"
+maintainer: ["Merlin04"]
+authors: ["Merlin04"]
```

Comment:
@mseri Just to confirm: does a github handle satisfy our policy for "a valid maintainer so that we can ping or send an email to them if needed"?

Reply:
It would be preferrable to have an email, if that is possible

Reply:
Thanks!

Are you ok with supplying an email address for contact here @Merlin04?

Reply:
Sure, I could set up a custom alias on my domain for this? I'm worried about spam but I think if I use a custom alias then if it gets picked up by any scrapers I could just disable it. Has this been an issue for anyone in the past?

Reply:
A custom alias is perfectly fine. I don't get different amount on spam on the address I use to sign my commits compared to the ones that I don't use on github, what I get is picked up from the spam filter and I don't really know which part is coming from being exposed here or from my emails being in some leak or some other website

Reply:
fixed!